plugin/src/test/groovy/com/novoda/test/TestProject.groovy

```diff
@@ -109,10 +120,18 @@ ${project.additionalConfiguration}
         projectDir.deleteDir()
     }
 
+    String projectDir(){
```

Comment:
I need to expose this to configure the `output` directory of the detekt plugin in the `DetektIntegrationTest `.